### PR TITLE
feat: add command to restart Gleam language server

### PR DESCRIPTION
**Pull request by cijiugechu**
close: #44

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -7,16 +7,41 @@ import {
...
+				await client.restart();
+
+        vscode.window.showInformationMessage("gleam server restarted.");
+			} else {
+				await client.start();
+			}
+		} catch (err) {
+			client.error("Restarting client failed", err, "force");
+		}
+  });
```
Comment (lpil) [inline src/extension.ts:41]: Could you run the prettier formatter on this file please? 🙏

Comment (cijiugechu) [reply src/extension.ts:41]: Should I add the Prettier dependency and corresponding configuration to the project? Future contributors may also need it.

Comment (lpil) [reply src/extension.ts:41]: If you want, I don't mind either way :)

**Review by lpil — APPROVED** [review]

**Comment by lpil** [comment]
Let me know if you'd like me to merge or wait for any more

**Comment by cijiugechu** [comment]
Thanks for your review, I believe this pull request is ready to be merged😊.

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
+  "semi": true,
+  "singleQuote": false,
+  "endOfLine": "lf",
+  "arrowParens": "always"
+}
```
Comment (lpil) [inline .prettierrc:7]: Could you remove this config please 🙏

Comment (cijiugechu) [reply .prettierrc:7]: Should the entire file be considered or just the three quoted lines?

Comment (lpil) [reply .prettierrc:7]: All the configuration, we'll use the defaults 👍

**Review by lpil — APPROVED** [review]
Thank you 💜

**Pull request merged**